### PR TITLE
Add description and group to vmware workflow variables

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

**Review by Hiba-Alili — APPROVED** [review]

File: ApplicationConnectors/resources/catalog/Centreon_Notification.xml [hunk 1]
```diff
@@ -283,9 +283,9 @@ QUERY_PARAM_<param_name> ]]>
...
         <variable inherited="false" name="RESPONSE_PATH" value="." description="Which data to extract in the response if json, xml or html format is selected. It uses the &lt;a href=&quot;https://groovy-lang.org/processing-xml.html&quot; target=&quot;_blank&quot;&gt;GPath notation&lt;/a&gt;" group="Http Response"/>
-        <variable inherited="false" name="HEADER_centreon-auth-token" value="${AUTH_TOKEN}"/>
-        <variable inherited="false" name="QUERY_PARAM_action" value="submit"/>
-        <variable inherited="false" name="QUERY_PARAM_object" value="centreon_submit_results"/>
+        <variable inherited="false" name="HEADER_centreon-auth-token" value="${AUTH_TOKEN}" description="A valid token for user authentication."/>
+        <variable inherited="false" name="QUERY_PARAM_action" value="submit" description="Parameter used to specify the action type."/>
```
Comment (nebilbenmabrouk) [inline ApplicationConnectors/resources/catalog/Centreon_Notification.xml:287]: I suggest: "Action type to be executed against Centreon" or "Action type to be executed in Centreon"

File: ApplicationConnectors/resources/catalog/Centreon_Notification.xml [hunk 2]
```diff
@@ -283,9 +283,9 @@ QUERY_PARAM_<param_name> ]]>
...
+        <variable inherited="false" name="QUERY_PARAM_object" value="centreon_submit_results" description="Parameter used to submit results to the monitoring engine."/>
```
Comment (nebilbenmabrouk) [inline ApplicationConnectors/resources/catalog/Centreon_Notification.xml:288]: I suggest: "Parameter used to specify the payload of the action to be executed against Centreon"

File: BigData/resources/catalog/Parse_Update_Json_From_Global_Space.xml [hunk 3]
```diff
@@ -3,12 +3,12 @@
...
   <variables>
-    <variable name="CLUSTER_NAME" value="activeeon-spark-cluster-${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CLUSTER_USER" value="act-usr" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CLUSTER_PASSWORD" value="" model="PA:HIDDEN"/>
-    <variable name="CLUSTER_SSH_USER" value="act-usr" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CLUSTER_SSH_PASSWORD" value="" model="PA:HIDDEN"/>
-    <variable name="JSON_FILE" value="" model="PA:GLOBAL_FILE"/>
+    <variable name="CLUSTER_NAME" value="activeeon-spark-cluster-${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING" description="Name of the Spark cluster to be created." group="Spark Cluster Parameters" advanced="false" hidden="false"/>
+    <variable name="CLUSTER_USER" value="act-usr" model="PA:NOT_EMPTY_STRING" description="User login that will be used to access the Web dashboard of the Spark cluster." group="Spark Cluster Parameters" advanced="true" hidden="false"/>
```
Comment (nebilbenmabrouk) [inline BigData/resources/catalog/Parse_Update_Json_From_Global_Space.xml:7]: Please avoid the future tense. I suggest "User login needed to access the Web dashboard of the Spark cluster."

File: BigData/resources/catalog/Parse_Update_Json_From_Catalog.xml [hunk 4]
```diff
@@ -3,11 +3,11 @@
...
   <variables>
-    <variable name="CLUSTER_NAME" value="activeeon-spark-cluster-${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CLUSTER_USER" value="act-usr" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CLUSTER_PASSWORD" value="" model="PA:HIDDEN"/>
-    <variable name="CLUSTER_SSH_USER" value="act-usr" model="PA:NOT_EMPTY_STRING"/>
-    <variable name="CLUSTER_SSH_PASSWORD" value="" model="PA:HIDDEN"/>
+    <variable name="CLUSTER_NAME" value="activeeon-spark-cluster-${PA_JOB_ID}" model="PA:NOT_EMPTY_STRING" description="Name of the Spark cluster to be created." group="Spark Cluster Parameters" advanced="false" hidden="false"/>
+    <variable name="CLUSTER_USER" value="act-usr" model="PA:NOT_EMPTY_STRING" description="User login that will be used to access the Web dashboard of the Spark cluster." group="Spark Cluster Parameters" advanced="true" hidden="false"/>
```
Comment (nebilbenmabrouk) [inline BigData/resources/catalog/Parse_Update_Json_From_Catalog.xml:7]: Please avoid the future tense. I suggest "User login needed to access the Web dashboard of the Spark cluster." Same for th others.

File: VMware/resources/catalog/Check_Service_${SERVICE_NAME}.xml [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
   <variables>
-    <variable name="SERVICE_NAME" value=""/>
-    <variable name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"/>
-    <variable name="VCENTER_USER" value=""/>
-    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value=""/>
-    <variable name="VCENTER_VM_NAME" value=""/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_USER" value="" description="Basic authentication user" group="VMware vCenter Connection"/>
+    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value="" description="Basic authentication password" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/Check_Service_${SERVICE_NAME}.xml:7]: "Name of the VM where the service (to be checked) is running"

Comment (nebilbenmabrouk) [inline VMware/resources/catalog/Check_Service_${SERVICE_NAME}.xml:7]: "Name of the VM where the service (to be checked) is running"

File: VMware/resources/catalog/Check_Service_${SERVICE_NAME}.xml [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
+    <variable model="" name="SERVICE_NAME" value="" description="The name of the service to check at the VM startup" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/Check_Service_${SERVICE_NAME}.xml:8]: "Name of the service to check"

File: VMware/resources/catalog/Get_VM_State_${VCENTER_VM_NAME}.xml [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
   <variables>
-    <variable model="" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"/>
-    <variable model="" name="VCENTER_USER" value=""/>
-    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value=""/>
-    <variable model="" name="VCENTER_VM_NAME" value=""/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_USER" value="" description="Basic authentication user" group="VMware vCenter Connection"/>
+    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value="" description="Basic authentication password" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/Get_VM_State_${VCENTER_VM_NAME}.xml:7]: "Name of the VM to check its status"

File: VMware/resources/catalog/Stop_VM_${VCENTER_VM_NAME}.xml [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
   <variables>
-    <variable model="" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"/>
-    <variable model="" name="VCENTER_USER" value=""/>
-    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value=""/>
-    <variable model="" name="VCENTER_VM_NAME" value=""/>
-    <variable name="STOP_ITERATION" value="600"/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_USER" value="" description="Basic authentication user" group="VMware vCenter Connection"/>
+    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value="" description="Basic authentication password" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/Stop_VM_${VCENTER_VM_NAME}.xml:7]: "Name of the VM to stop"

File: VMware/resources/catalog/VCENTER_Get_VM_ID_By_Name.xml [hunk 9]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <job xmlns="urn:proactive:jobdescriptor:3.13" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" maxNumberOfExecution="2" name="VCENTER_Get_VM_ID_By_Name" onTaskError="continueJobExecution" priority="normal" projectName="2. Rest Simple Tasks" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd">
   <variables>
-    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"/>
-    <variable model="" name="VCENTER_VM_NAME" value=""/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Get_VM_ID_By_Name.xml:5]: "Name of the VM for which to get the ID"

File: VMware/resources/catalog/VCENTER_Get_VM_State.xml [hunk 10]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <job xmlns="urn:proactive:jobdescriptor:3.13" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" maxNumberOfExecution="2" name="VCENTER_Get_VM_State" onTaskError="continueJobExecution" priority="normal" projectName="2. Rest Simple Tasks" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd">
   <variables>
-    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_ID" value=""  description="The identifier of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Get_VM_State.xml:5]: "The identifier of the VM to check its status"

File: VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml [hunk 11]
```diff
@@ -1,16 +1,16 @@
...
-    <variable model="PA:URL" name="CENTREON_ENDPOINT" value="https://centreon.xxx.com/centreon/api"/>
-    <variable name="CENTREON_NOTIF_HOST" value=""/>
-    <variable name="CENTREON_NOTIF_SERVICE" value=""/>
-    <variable name="CENTREON_USER" value=""/>
-    <variable name="CENTREON_PASSWORD" value=""/>
-    <variable name="SERVICE_NAME" value=""/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_USER" value="" description="Basic authentication user" group="VMware vCenter Connection"/>
+    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value="" description="Basic authentication password" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml:7]: "The name of the VM to restart"

File: VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml [hunk 12]
```diff
@@ -1,16 +1,16 @@
...
+    <variable model="" name="SERVICE_NAME" value="" description="The name of the service to check at the VM startup" group="VMware vCenter parameters"/>
+    <variable model="PA:URL" name="CENTREON_ENDPOINT" value="https://centreon.xxx/centreon/api"  description="Centreon server base URL" group="Centreon Connection"/>
+    <variable model="" name="CENTREON_USER" value="" description="The user who will authenticate to Centreon server and submit results"  group="Centreon Connection"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml:10]: "User login needed to connect to Centreon server"

File: VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml [hunk 13]
```diff
@@ -1,16 +1,16 @@
...
+    <variable model="" name="CENTREON_PASSWORD" value="" description="The user password"  group="Centreon Connection"/>
+    <variable model="" name="CENTREON_NOTIF_HOST" value=""  description="The host from which the notification is sent" group="Centreon Parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml:12]: "The host from which the notification is sent to Centreon"

File: VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml [hunk 14]
```diff
@@ -1,16 +1,16 @@
...
+    <variable model="" name="CENTREON_NOTIF_SERVICE" value="" description="The name of the notification service in the server side" group="Centreon Parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Restart_VM_${VCENTER_VM_NAME}.xml:13]: "Name of the Centreon service that will receive the notification"

File: VMware/resources/catalog/VCENTER_Shutdown_VM.xml [hunk 15]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <job xmlns="urn:proactive:jobdescriptor:3.13" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" maxNumberOfExecution="2" name="VCENTER_Shutdown_VM" onTaskError="continueJobExecution" priority="normal" projectName="2. Rest Simple Tasks" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd">
   <variables>
-    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_ID" value=""  description="The identifier of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Shutdown_VM.xml:5]: "The identifier of the VM to shutdown"

File: VMware/resources/catalog/VCENTER_Shutdown_VM_${VCENTER_VM_NAME}.xml [hunk 16]
```diff
@@ -1,15 +1,15 @@
...
-    <variable model="" name="VCENTER_VM_NAME" value=""/>
-    <variable model="PA:URL" name="CENTREON_ENDPOINT" value="https://centreon.xxx/centreon/api"/>
-    <variable name="CENTREON_NOTIF_HOST" value="SRV-SCHEDULER"/>
-    <variable model="" name="CENTREON_NOTIF_SERVICE" value=""/>
-    <variable model="" name="CENTREON_USER" value=""/>
-    <variable model="" name="CENTREON_PASSWORD" value=""/>
+    <variable model="PA:URL" name="VCENTER_ENDPOINT" value="https://xxx-vcenter/rest"  description="VMware vCenter server base URL" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_USER" value="" description="Basic authentication user" group="VMware vCenter Connection"/>
+    <variable model="PA:HIDDEN" name="VCENTER_PASSWORD" value="" description="Basic authentication password" group="VMware vCenter Connection"/>
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Shutdown_VM_${VCENTER_VM_NAME}.xml:7]: "Name of the VM to shutdown"

File: VMware/resources/catalog/VCENTER_Shutdown_VM_${VCENTER_VM_NAME}.xml [hunk 17]
```diff
@@ -1,15 +1,15 @@
...
+    <variable model="PA:URL" name="CENTREON_ENDPOINT" value="https://centreon.xxx/centreon/api"  description="Centreon server base URL" group="Centreon Connection"/>
+    <variable model="" name="CENTREON_USER" value="" description="The user who will authenticate to Centreon server and submit results"  group="Centreon Connection"/>
+    <variable model="" name="CENTREON_PASSWORD" value="" description="The user password"  group="Centreon Connection"/>
+    <variable model="" name="CENTREON_NOTIF_HOST" value=""  description="The host from which the notification is sent" group="Centreon Parameters"/>
+    <variable model="" name="CENTREON_NOTIF_SERVICE" value="" description="The name of the notification service in the server side" group="Centreon Parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Shutdown_VM_${VCENTER_VM_NAME}.xml:12]: Same as above

File: VMware/resources/catalog/VCENTER_Start_VM_${VCENTER_VM_NAME}.xml [hunk 18]
```diff
@@ -1,16 +1,16 @@
...
+    <variable model="" name="VCENTER_VM_NAME" value="" description="The name of the VM to start" group="VMware vCenter parameters"/>
+    <variable model="" name="SERVICE_NAME" value="" description="The name of the service to check at the VM startup" group="VMware vCenter parameters"/>
+    <variable model="PA:URL" name="CENTREON_ENDPOINT" value="https://centreon.xxx/centreon/api"  description="Centreon server base URL" group="Centreon Connection"/>
+    <variable model="" name="CENTREON_USER" value="" description="The user who will authenticate to Centreon server and submit results"  group="Centreon Connection"/>
+    <variable model="" name="CENTREON_PASSWORD" value="" description="The user password"  group="Centreon Connection"/>
+    <variable model="" name="CENTREON_NOTIF_HOST" value=""  description="The host from which the notification is sent" group="Centreon Parameters"/>
+    <variable model="" name="CENTREON_NOTIF_SERVICE" value="" description="The name of the notification service in the server side" group="Centreon Parameters"/>
```
Comment (nebilbenmabrouk) [inline VMware/resources/catalog/VCENTER_Start_VM_${VCENTER_VM_NAME}.xml:13]: Same as above

**Review by nebilbenmabrouk — APPROVED** [review]

**Pull request merged**